File: hw/medina/u1/HW07ChickenFarmSimulatorWithImput/newChickenJS/model/Egg.js
/**
 * Egg class
 * Represents an egg laid by a chicken.
 */
class Egg {
  /**
   * @param {string} size
   */
  constructor(size) {
    this.size = size;
  }

  toString() {
    return "Egg{ size = " + this.size + " }";
  }

  getSize() {
    return this.size;
  }
}

module.exports = { Egg };

File: ws/gudino/u1/ws05CodingClasses/ChikenFarmSimulator/src/ec/edu/espe/chikenfarmsimulator/model/Egg.java
package ec.edu.espe.chikenfarmsimulator.model;

/**
 *
 * @author Bryan Gudino, KNOWLEDGE ENCAPSULATE, @ESPE
 */
public class Egg {
    private char size;

    @Override
    public String toString() {
        return "Egg{" + "size=" + getSize() + '}';
    }

    public Egg(char size) {
        this.size = size;
    }

    /**
     * @return the size
     */
    public char getSize() {
        return size;
    }

    /**
     * @param size the size to set
     */
    public void setSize(char size) {
        this.size = size;
    }
}
